mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from "src/component/select/Select.module.scss";
+
+/**
+ * Implement an option element for a select component
+ */
+export interface OptionType {
+  /**
+   * Unique value
+   */
+  id: string;
+  /**
+   * Option value
+   */
+  value: string;
+  /**
+   * Option`s text
+   */
+  text: string;
+}
+
+/**
+ * Implement a simple select component
+ */
+export interface SelectProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default select value
+   */
+  value: string;
+  /**
+   * Select's name
+   */
+  name: string;
+  /**
+   * Options list
+   */
+  options: OptionType[];
+  /**
+   * Default value for select
+   */
+  selected: string;
```

Comment:
why should we have default `value` and `selected`? Why not to use default option value or id?

Reply:
we can use for example options[0].value then i remove selected from props.

Reply:
isn`t it?

Reply:
done

Reply:
Can you dont use properties selected and  defaultvalue?